app/views/student_requests/adminview.html.haml

```diff
@@ -21,47 +23,56 @@
+      %thead 
         %tr
-          %td
-            %a{:onclick => "return buttonclick(#{record.uin})"} #{record.uin}
-          %td= record.full_name
-          %td= record.major
-          %td= record.classification
-          %td= record.minor
-          %td= record.course_id
-          %td= record.section_id
-          %td= record.state
-          %td= record.priority
-          %td= record.notes
-          %td
-            %center
-            = form_tag('/student_requests/updaterequestbyadmin', method: "put") do
-              = hidden_field_tag 'id', record.request_id
-              %textarea{:name => "notes_for_student", :placeholder => "Notes for student", :class => "notes"}
-              %textarea{:name => "notes_for_myself", :placeholder => "Notes for Myself", :class => "notes"}
-              = select_tag("state", options_for_select(@allAdminStates))
-              = select_tag("priority", options_for_select(@allPriorityStates))
-              = submit_tag 'Update'
-          %td= mail_to record.email, "email", subject: "Force Request System: Regarding course #{record.course_id}" 
+          %th 
+          %th UIN
+          %th Full Name
+          %th Major
+          %th Class
+          %th Minor
+          %th Course ID
+          %th Section ID
+          %th State
+          %th Priority
+          %th Notes
+          %th{ :class => "update_admin" } Action
+          %th Email
+      
+      %tbody
+        :javascript
+          var email_id_list = [];
+        - @coursestudentlist[course].each do |record|
+          %tr
+            %td= check_box_tag "request_ids[]", record.request_id, false, :onclick => "email_id_list.push(\"#{record.email}\")"
+            %td
+              %a{:onclick => "return buttonclick(#{record.uin})"} #{record.uin}
+            %td= record.full_name
+            %td= record.major
+            %td= record.classification
+            %td= record.minor
+            %td= record.course_id
+            %td= record.section_id
+            %td= record.state
+            %td= record.priority
+            %td= record.notes
+            %td
+              %center
+              = form_tag('/student_requests/updaterequestbyadmin', method: "put") do
+                = hidden_field_tag 'id', record.request_id
+                %textarea{:name => "notes_for_student", :placeholder => "Notes for student", :class => "notes"}
+                %textarea{:name => "notes_for_myself", :placeholder => "Notes for Myself", :class => "notes"}
+                = select_tag("state", options_for_select(@allAdminStates))
+                = select_tag("priority", options_for_select(@allPriorityStates))
+                = submit_tag 'Update'
+            %td= mail_to record.email, "email", subject: "Force Request System: Regarding course #{record.course_id}" 
+            
+    = select_tag("multi_state_sel", options_for_select(@allAdminStates))
```

Comment:
Right now, you are only updating state. I think it would be great if you can do the same for priority. To make it generalized enough.